examples/cifar10_resnet.py

```diff
@@ -237,30 +248,29 @@ def resnet_v2(input_shape, depth, num_classes=10):
     if (depth - 2) % 9 != 0:
         raise ValueError('depth should be 9n+2 (eg 56 or 110 in [b])')
     # Start model definition.
-    inputs = Input(shape=input_shape)
     num_filters_in = 16
-    num_filters_out = 64
-    filter_multiplier = 4
-    num_sub_blocks = int((depth - 2) / 9)
+    num_res_units = (depth - 2) / 9
```

Comment:
Should be // instead of /
/ is doing integer division only for Python2

Reply:
Thank you. I think I'll put `int()` back instead to avoid gratuitous changes. `//` (88) and `int()` (68) are used approximately equally in Keras.